components/admin_console/data_table/data_table.scss

```diff
@@ -0,0 +1,119 @@
+.Table__table {
+    width: 100%;
+
+    th, td {
+        padding: 4px 12px;
+    }
+}
+
+.Table__table-header {
+    line-height: 16px;
+    font-weight: 600;
+    border-bottom: 1px solid rgba(var(--sys-center-channel-color-rgb), 0.16);
+
+    &:last-child {
+        border-bottom: 1px solid rgba(var(--sys-center-channel-color-rgb), 0.08);
+    }
+}
+
+.Table__table-row {
```

Comment:
Kind of have some smurf naming going on here: https://devcards.io/smurf-naming-convention